### ISSUE TITLE
Code Parallelization

### ISSUE THREAD

**Issue by pcosme** [pcosme/TETHYS-Graphene-Hydrodynamic-Simulation#30]
In order to shorten the computation time in 2D simulation parallelization solutions must be implemented. 
For an east start we should determine which processes can run independently and in parallel to be distributed to several CPU cores.

**Comment by pcosme** [comment]
I'll postpone this analysis for a latter time
